processing-directory-english.adoc

```diff
@@ -38,11 +38,13 @@ Name, form of address, title, password (encrypted), email addresses, invoice add
 
 === Recipient categories
 
-Payment service providers, shipping service provider, email service of the seller, AWS
+Payment service providers, shipping service provider, email service of the seller, Amazon Web Services
 
 === Data transfer to third countries
 
-No data transfer outside the European Union.
+Secure third countries: Canada, Switzerland, USA
+
+Other third countries (depending on the marketplace): Australia, Hong Kong, India, Malaysia, Philippines, Singapore
```

Comment:
Ist das in Ordnung so oder zu knapp?

Reply:
Mein Vorschlag:
---
 === Data transfer to third countries

depending on the marketplace:
No data transfer outside the European Union

OR

Secure third countries: Canada, Switzerland, USA

Other third countries: Australia, Hong Kong, India, Malaysia, Philippines, Singapore
---

@awinzek wie passt dies zum Stil?

Reply:
@snawratil , in Absprache mit @vminhta finden wir die folgende Beschreibung am besten. Bitte auch  – falls du das so übernimmst – in der DE-Datei anpassen (der korrekte Begriff für adequacy decision ist übrigens Angemessenheitsentscheidung).



=== Data transfer to third countries

Depending on the marketplace, there is either no data transfer outside the European Union or there is data transfer:

* Currently secure third countries (according to the adequacy decision of the European Commission): Canada, Switzerland, USA

* Other third countries: Australia, Hong Kong, India, Malaysia, Philippines, Singapore